warehouse/query-core/src/main/java/datawave/query/composite/CompositeTerm.java

```diff
@@ -85,7 +89,20 @@ public void getNodesAndExpressions(List<Class<? extends JexlNode>> nodeClasses,
      * @return
      */
     private String getAppendedExpressions() {
-        return String.join(separator, expressionList);
+        final List<String> newExpressions;
+        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && !jexlNodeList.stream().allMatch(x -> x instanceof ASTERNode)) {
+            // We have an ASTERNode in the last position, but the rest are not ERNodes ( and therefore must be EQNodes )
+            // escape all of the terms
+            newExpressions = expressionList.stream().map(term -> {
+                return Pattern.quote(term);
```

Comment:
```suggestion
            newExpressions = expressionList.stream().map(Pattern::quote).collect(Collectors.toList());
```

Reply:
I thought I changed this to a for loop....I must have messed up my commit.

Reply:
Oh well I'm not sure why I switched to a for loop. I didn't stash it. Maybe I just accepted this as shorter. I think I just disliked the idea of replacing the last node.